android/src/main/java/com/tailscale/ipn/ui/viewModel/IpnViewModel.kt

```diff
@@ -70,17 +68,11 @@ open class IpnViewModel : ViewModel() {
   }
 
   fun startVPN() {
-    val context = App.getApplication().applicationContext
-    val intent = Intent(context, IPNReceiver::class.java)
-    intent.action = IPNReceiver.INTENT_CONNECT_VPN
-    context.sendBroadcast(intent)
+    App.getApplication().startVPN()
```

Comment:
We can reuse the applications stop and start methods instead of building intents by hand.